src/sentry/static/sentry/app/views/performance/charts/chart.tsx

```diff
@@ -16,6 +19,11 @@ type Props = {
   loading: boolean;
 };
 
+function roundAxis(x) {
+  const exp10 = 10 ** Math.floor(Math.log10(x));
+  return Math.ceil(x / exp10) * exp10;
+}
```

Comment:
Couldn't find if/where echart exposes their rounding for the axis max value, but worst case for this when it rounds 10001 -> 20000 leaving half the chart unused.

Reply:
Won't that rounding impact all values between 10001->18000? Would something like

```javascript
exp = Math.max(1, Math.floor(Math.log10(x)) - 1)
return Math.ceil(x) + 10 ** exp
```

Give us smaller gaps at the top?

Reply:
after a little research, i stumbled across https://stackoverflow.com/questions/11397239/rounding-up-for-a-graph-maximum and adapted it for our use case